analyzers/tests/SonarAnalyzer.UnitTest/SymbolicExecution/Roslyn/RoslynSymbolicExecutionTest.Nullable.cs

```diff
@@ -54,7 +54,7 @@ public void Nullable_Value_ReadsConstraintsFromInstance()
             """;
         var setter = new PreProcessTestCheck(OperationKind.Literal, x => x.Operation.Instance.ConstantValue.Value is false ? x.SetOperationConstraint(TestConstraint.First) : x.State);
         var validator = SETestContext.CreateCS(code, ", bool? arg", setter).Validator;
-        validator.ValidateTag("Unknown", x => x.Should().BeNull());
+        validator.ValidateTag("Unknown", x => x.HasConstraint(ObjectConstraint.NotNull).Should().BeTrue("Accessing .Value would already throw, so it is NotNull by now"));
```

Comment:
This tests `value` which must be a non-nullabel value type. The assertion isn't of much value.
But `arg` should also be known to be NotNull here. Add an assertion for `arg`. Also move the "because" to the new assertion.